src/api/discord/upload.js

```diff
@@ -42,7 +42,7 @@ async function upload(filePath, title) {
         await thread.join();
         // Send a confirmation message
         await thread.send(`Created and joined thread: ${thread.name}`);
-        await thread.send(filePath); // yet add attachment support
+        await thread.send({ files: [filePath] }); // yet add attachment support
```

Comment:
From where the shit are you getting `filePath` man? I don't see any variable declared.

Reply:
Check the method declaration, it'll be passed as a param